sdk/documentintelligence/ai-document-intelligence-rest/samples-dev/composeModel.ts

```diff
@@ -77,8 +77,7 @@ async function main() {
       description: "A composed model that classifies purchase order documents and extracts data from them.",
       componentModels: modelIds,
       modelId: composedModelId,
-
-    },
+    } as any,
```

Comment:
is it possible to cast to the expected type of `client.path("/documentModels:compose").post()`?

Reply:
This is a bug in the emitted type and there is work to fix it. /cc @HarshaNalluru

Reply:
https://github.com/Azure/autorest.typescript/issues/2127

Reply:
wait.. why is this casted.. the issue logged in was for Line 89 in this file

Reply:
Hmm, I think there was some miscommunication. There is currently a type error at this line and that was the reason I added this cast:
![image](https://github.com/user-attachments/assets/294e243b-52f2-496b-bd52-401d00264e46)

Reply:
this is not expected, maybe the sample is out of date, I need to check